src/proxy/ClientBuild.php

```diff
@@ -71,41 +71,63 @@ public function init()
     }
     
     private $_buildConfig;
-    
+
     public function setBuildConfig(array $config)
     {
         $this->_buildConfig = $config;
-        
+
         foreach ($config['tables'] as $tableName => $tableConfig) {
             if (!empty($this->optionTable)) {
-                $skip = true;
-                
-                foreach ($this->optionTable as $useName) {
-                    if ($useName == $tableName || StringHelper::startsWithWildcard($tableName, $useName)) {
-                        $skip = false;
-                    }
-                }
-                
+
+                $skip = $this->skipTable($tableName);
                 if ($skip) {
                     continue;
                 }
             }
-            
+
             $schema = Yii::$app->db->getTableSchema($tableName);
-            
+
             if ($schema !== null) {
                 $this->_tables[$tableName] = new ClientTable($this, $tableConfig);
             }
         }
     }
-    
+
+    /**
+     * @param $tableName
+     * @return bool
+     *
+     * @since  1.2.1
+     */
+    private function skipTable($tableName): bool
```

Comment:
We don't use php 7 output hinting yet and it could be an useful method for others so i would make it public or is there a specific reason to make it private?

Reply:
As mentioned before i would rename to isSkipableTable as its more clear it will return a boolean value whether its skipable or not. 

I also recommend to provide the lookup tables array as an argument, this makes it more easy to test and maintanable. So final method could look like this:

```php
public function isSkipableTable($tableName, array $tables)
{

}
```